.github/build-x86_64-apple-darwin

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+echo ✨ System Info  
+sw_vers
+echo ✨ System Details
+uname -a
+echo ✨ LLVM Version
+llvm-config --version
+
+echo ✨ CMAKE_CXX_COMPILER=$CMAKE_CXX_COMPILER
+echo ✨ CMAKE_C_COMPILER=$CMAKE_C_COMPILER
+
+# llvm@11 paths

```

Comment:
Do we require llvm 11, is it the oldest llvm we support, or is the choice arbitrary?

Reply:
Barretenberg does not specify clang version afaik other than this [README.md](https://github.com/AztecProtocol/barretenberg/tree/master/barretenberg#dependencies) recommendation which states clang 9 or newer. Some build tools seem to be relying on [clang 10](https://github.com/AztecProtocol/barretenberg/blob/b3de08413473e2ef7650db17c2427b9cddf0dd4a/barretenberg/dockerfiles/Dockerfile.x86_64-linux-clang#L11) 

GH macos-latest runner brings Clang/LLVM 13.0.0 as default which does not work. Clang/LLVM 11 was found as the latest working in that specific setup. Also, Clang/LLVM 11 comes as default on some distros so it was settled on for consistency here.